src/FSAL/FSAL_CORTXFS/main.c

```diff
@@ -73,8 +73,8 @@ static const struct fsal_staticfsinfo_t default_kvsfs_info = {
 					   it is possible to cross junctions
 					   for resolving an NFS export path. */
 	.delegations = FSAL_OPTION_FILE_DELEGATIONS,	/*< fsal supports delegations */ /* TODO */
-	.pnfs_mds = false,		/*< fsal supports file pnfs MDS */ /* TODO */
-	.pnfs_ds = false,		/*< fsal supports file pnfs DS */ /* TODO */
+	.pnfs_mds = true,		/*< fsal supports file pnfs MDS */ /* TODO */
+	.pnfs_ds = true,		/*< fsal supports file pnfs DS */ /* TODO */
```

Comment:
Doubt: Don't we need to keep pNFS disabled by default?

Reply:
Since the CORTX FSAL is expected to support pNFS i thought we can leave this enabled by default.But the per export flag(pnfs_mds_enabled flag in kvsfs_fsal_export) will be controlled via the pNFS role flag that is parsed from pNFS.conf.But if you will we should control this flag also via pNFS role, i will make that change

Reply:
In my opinion, the pNFS role is user controlled settings via config file, which should be always at a higher priority. But in case that role is not provided (i.e. default is disabled), then the default value of this global structure should also be similar.

Reply:
@pratyush-seagate , made changes as per your input